cli/install/cni-template.go

```diff
@@ -18,19 +18,65 @@
 
 package install
 
-// CNITemplate provides the base template for the `linkerd install-cni-plugin` command.
-const CNITemplate = `### Namespace ###
+const (
+	// CNITemplate provides the base template for the `linkerd install-cni-plugin` command.
+	CNITemplate = `### Namespace ###
 kind: Namespace
 apiVersion: v1
 metadata:
   name: {{.Namespace}}
 ---
+apiVersion: policy/v1beta1
```

Comment:
Hmm, we should move this all into the helm chart at some point.

Reply:
Added https://github.com/linkerd/linkerd2/issues/292.